### PR TITLE
refactor(components): update Box stories

### PR DESCRIPTION
**Pull request by koji**
# Overview
update Box stories

close AUTH-294

# Test Plan

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

File: components/src/primitives/Box.stories.tsx [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-  border: '1px solid black',
-  padding: '1rem',
-  maxWidth: '20rem',
+export const Box: Story = {
+  args: {
+    children:
+      'This is a simple box atom that accepts all primitive styling props.',
+    backgroundColor: 'grey',
+    border: '1px solid black',
+    padding: '1rem',
```
Comment (jerader) [inline components/src/primitives/Box.stories.tsx]: do we want to use the constant here?

File: components/src/primitives/Box.stories.tsx [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
-    'This is a simple box atom that accepts all primitive styling props.',
-  backgroundColor: 'grey',
-  border: '1px solid black',
-  padding: '1rem',
-  maxWidth: '20rem',
+export const Box: Story = {
+  args: {
+    children:
+      'This is a simple box atom that accepts all primitive styling props.',
+    backgroundColor: 'grey',
```
Comment (jerader) [inline components/src/primitives/Box.stories.tsx]: should we use a color in our color system?

File: components/src/primitives/Box.stories.tsx [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
+    border: '1px solid black',
```
Comment (jerader) [inline components/src/primitives/Box.stories.tsx]: same here, we can use a border constant

**Review by jerader — APPROVED** [review]
lgtm but left comments to use some typography constants

File: components/src/primitives/Box.stories.tsx (hunk 1 above)
Comment (koji) [reply components/src/primitives/Box.stories.tsx]: i will update.

File: components/src/primitives/Box.stories.tsx (hunk 2 above)
Comment (koji) [reply components/src/primitives/Box.stories.tsx]: yes, i will update.

**Pull request merged**